### PR TITLE
add features tab to main readme.

### PR DESCRIPTION
**Pull request by ahmetoz**

### CODE REVIEW

**Review by lojzatran — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
+
+- [Features](#features)
 - [Overview](#overview)
 - [Extension module](#extension-module)
 - [Notification module](#notification-module)
 - [Contribution Guide](#contribution-guide)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Features
```
Comment (butenkor) [inline README.md:18]: Maybe rename it to `Supported features` then we can omit below the repetitive `Integration supports`?

File: README.md [hunk 2]
```diff
@@ -1,20 +1,31 @@
...
+
+- Integration supports all the payment methods available as an [Adyen Web Component](https://docs.adyen.com/checkout/components-web). For a full list of all supported web component based payment methods please refer to [supported payment methods](https://docs.adyen.com/checkout/supported-payment-methods).
+  - Note: since the integration relies on the usage of Adyen's web components it does not need to process sensitive credit card data and thus is fully PCI DSS **compliant**.
+- Integration supports asynchronous notifications handling via [Notification module](#notification-module). 
+- Integration supports [multi-tenancy](./extension/docs/WebComponentsIntegrationGuide.md#multi-tenancy) to serve multiple Adyen merchant accounts/commercetools projects with one application instance.
+- Integration supports [refunding](./extension/docs/Refund.md) a payment back to the shopper.
+- Integration supports [cancelling](./extension/docs/CancelPayment.md) the authorisation on a payment that has not yet been captured.
```
Comment (butenkor) [inline README.md:25]:
```suggestion
- Integration supports authorisation [cancellation](./extension/docs/CancelPayment.md) the on a payment that has not yet been captured.
```

File: README.md [hunk 3]
```diff
@@ -1,20 +1,31 @@
...
+- Integration supports [manual capturing](./extension/docs/ManualCapture.md) of a payment.
```
Comment (butenkor) [inline README.md:26]:
```suggestion
- Integration supports [manual capture](./extension/docs/ManualCapture.md) of a payment.
```

File: README.md [hunk 4]
```diff
@@ -1,20 +1,31 @@
...
+- Integration supports [restore](./extension/docs/Restore.md) feature of Adyen, which gives your shoppers an opportunity to offset their carbon emissions from the delivery or lifecycle of their purchase at checkout.
```
Comment (butenkor) [inline README.md:27]:
```suggestion
- Integration supports [restore](./extension/docs/Restore.md), which gives your shoppers an opportunity to offset their carbon emissions from the delivery or lifecycle of their purchase at checkout. 
```

File: README.md (hunk 1 above)
Comment (ahmetoz) [reply README.md]: https://github.com/commercetools/commercetools-adyen-integration/pull/863/commits/6b36b219fa6682dd2f95dcc12e0b056360bfd2b4

File: README.md (hunk 2 above)
Comment (ahmetoz) [reply README.md]: https://github.com/commercetools/commercetools-adyen-integration/pull/863/commits/b4b31edf60e1118dd1224dd1089094cc4dccf050

File: README.md (hunk 4 above)
Comment (ahmetoz) [reply README.md]: https://github.com/commercetools/commercetools-adyen-integration/pull/863/commits/61e45345305551056273482e479b1ece6e4cb1eb

**Review by butenkor — APPROVED** [review]

**Pull request merged**